openmc/statepoint.py

```diff
@@ -501,6 +509,19 @@ def sparse(self, sparse):
             for tally_id in self.tallies:
                 self.tallies[tally_id].sparse = self.sparse
 
+    def add_volume_information(self, volume_calc):
+        """Add volume information to the geometry within the file
+
+        Parameters
+        ----------
+        volume_calc : openmc.VolumeCalculation
+            Results from a stochastic volume calculation
+
+        """
+        if self.summary is not None:
+            self.summary.add_volume_information(volume_calc)
+
+
```

Comment:
Very minor, but per PEP 8 style guide, one line after class methods unless you are separating groups of methods.